cpg-concepts/src/integrationTest/kotlin/de/fraunhofer/aisec/cpg/concepts/MemoryTest.kt

```diff
@@ -115,7 +280,7 @@ class MemoryTest {
                 )
             }
         assertNotNull(tree)
-        assertEquals(true, tree.value)
-        assertEquals(2, tree.children.size)
+        /*assertEquals(true, tree.value)
```

Comment:
can we reactivate these asserts?

Reply:
They should be part of the individual tests

Reply:
Btw. there are positive and negative tests now